scripts/build-testing-sdk.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Oh, just because it makes it an executable script. You don't have to invoke node. It's a convenience, just to make scripts executable programs, rather than an argument to an interpreter. Removing the `.js` isn't something I'm strongly opinionated about. I have vim set up to understand shebangs and set the syntax language, but if it's a pain for other editors, leave it on 😄